erc20-load-test-tool/script/run-dapp.py

```diff
@@ -0,0 +1,238 @@
+def aggregate_report(instance):
+    port= 8080
+    filename = "../output/result/report-"
+    aggregate_throughput = 0
+    aggregate_latency = 0
+    aggregate_tx = 0
+    aggregate_loadfactor = 0
+    aggregate_txStatus = {}
+    aggregate_txErrors = {}
+    aggregate_receiptStatus = {}
+    aggregate_receiptErrors = {}
+    for i in range(1, instance+1):
+        with open(filename+str(port+i)+'.json', 'r') as f:
+            data = json.load(f)
+            aggregate_tx += data['txTotal']
+            aggregate_throughput += data['averageThroughput']
+            aggregate_latency += data['averageLatency']
+            aggregate_loadfactor += data['loadFactor']
+
+            txStatus_list = data["txStatus"].split(",")
+            list_to_kv(txStatus_list, aggregate_txStatus)
+            
+            if(data["txErrors"]):
+                txErrors_list = data["txErrors"].split(",")
+                list_to_kv(txErrors_list, aggregate_txErrors)
+
+            if(data["receiptStatus"]):
+                receiptStatus_list = data["receiptStatus"].split(",")
+                list_to_kv(receiptStatus_list, aggregate_receiptStatus)
+            
+            if(data["receiptErrors"]):
+                receiptErrors_list = data["receiptErrors"].split(",")
+                list_to_kv(receiptErrors_list, aggregate_receiptErrors)
+                
+    json_obj = {}
+    json_obj['aggregate_tx'] = aggregate_tx
+    json_obj['aggregate_throughput'] = aggregate_throughput
+    json_obj['aggregate_latency'] = int(aggregate_latency/instance)
+    json_obj['aggregate_loadfactor'] = aggregate_loadfactor
+    json_obj['aggregate_txStatus'] = aggregate_txStatus
+    json_obj['aggregate_txErrors'] = aggregate_txErrors
+    json_obj['aggregate_receiptStatus'] = aggregate_receiptStatus
+    json_obj['aggregate_receiptErrors'] = aggregate_receiptErrors
+
+    filename = "../output/result/aggregate-report.json"
+    with open(filename, 'w') as f:
+        json.dump(json_obj, f, indent=4)
+
+
+def main():
+    instance = int(os.getenv('DAPP_INSTANCES', 1))
+    print("No of dapp Instances ",instance)
+
+    if(not os.getenv('WEB3J_ETHCLIENT_HOST')):
+        exit("Plz set WEB3J_ETHCLIENT_HOST env variable")
+
+    ethrpcApiUrl = os.getenv('WEB3J_ETHCLIENT_PROTOCOL', "http") + "://" + os.getenv(
+        'WEB3J_ETHCLIENT_HOST') + ":" + os.getenv('WEB3J_ETHCLIENT_PORT', "8545")
+    print("Ethereum Endpoint ",ethrpcApiUrl)
+
+    share_contract = os.getenv("SHARE_CONTRACT", 'False').lower() in ('true', '1', 't')
+
+    accts = []
+    priv_keys = []
+    for i in range(instance+1):
+        acct = w3help.eth.account.create('KEYSMASH FJAFJKLDSKF7JKFDJ 1530')
+        accts.append(Web3.toChecksumAddress(acct.address[2:].lower()))
+        priv_keys.append(acct.privateKey.hex()[2:].lower())
+
+    print("Account address list = ", accts)
+
+    threads = []
+    port = 8080
+    if(share_contract and instance>1):
```

Comment:
Sharing contract without multiple instances doesn't make sense, so let's assert that upfront.
`if share_contract:`